Hi @tomas-samot, thanks for your feedbacks. Sorry if it's a bit messy, I'll clean it and make a PR again. 
Without this :
```
libfreetype6-dev
libjpeg62-turbo-dev
libpng12-dev
```
I have some error for avatars display like `configure: error: jpeglib.h not found.`
I simply adapted [the original code proposed by the initial repo](https://github.com/PrivateBin/PrivateBin/commit/f45367fac2b5b9405c23f6867c551962122bc970).
`zip` doesn't seem be required, I will delete this line.

--- a/Dockerfile
+++ b/Dockerfile
@@ -10,19 +10,16 @@ RUN chmod a+x /cert.sh
 
 RUN sed -i "/#ServerName www.example.com/c\ServerName $DOMAIN" /etc/apache2/sites-available/000-default.conf
 
-RUN apt-get update && apt-get install -y \
+RUN apt-get update
+RUN apt-get install -y \
         libfreetype6-dev \
         libjpeg62-turbo-dev \
         libpng12-dev \
         wget \
-        zip \
-        unzip; \
-    # php-gd config
-    docker-php-ext-configure gd --with-freetype-dir=/usr/include/ --with-jpeg-dir=/usr/include/; \
-docker-php-ext-install -j$(nproc) gd;
+        unzip
 
-RUN apt-get install -y wget
-RUN apt-get install -y unzip
+RUN docker-php-ext-configure gd --with-freetype-dir=/usr/include/ --with-jpeg-dir=/usr/include/
+RUN docker-php-ext-install -j$(nproc) gd
 
 RUN set -x \
   && cd /var/www/html \

--- a/README.md
+++ b/README.md
@@ -14,6 +14,7 @@ Replace `example.com` with the domain you wish to use. (Dockerfile and `cert.sh`
 `sudo docker run -p 80:80 -p 443:443 -i -d -t --name PrivateBin -h example.com privatebin`
 
 ### Read and Run `cert.sh`
+`docker exec -i PrivateBin bash -c "cd / && bash cert.sh"`
 
 ### Add to Cron
 Remember to choose a different time

